src/main/resources/config/liquibase/changelog/20230908050001_changelog.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="zbil" id="20230908050001">
+        <createTable tableName="course_code_of_conduct_agreement">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="course_id" type="bigint"/>
+            <column name="user_id" type="bigint"/>
+            <column name="is_accepted" type="boolean"/>
```

Comment:
Maybe you can also add a constraint with nullable="false" and also set a default value of false.

Reply:
I revised it. I assume BIT cannot be NULL :) 1a7e7089c96d9cddb5cb328e82e3f0a5a25a500e